### DISCUSSION TITLE
Using AudioGridder with external remote server?

### DISCUSSION THREAD

**Discussion by WilliamKWusik** [apohl79/audiogridder#670] [General]
Is it possible to use it with an external remote server over the internet? I don't care about added latency as I would use it only for mixing and mastering. ;-)

Thanks!  <3

Cheers

**Comment by apohl79** [comment]
Yes, I don't see why that should not work.

**Comment by WilliamKWusik** [comment]
So far no luck, can't connect, no error message, just won't connect. I guess I need to setup things in Windows Firewall and also on my Router's Firewall...

Wk

**Reply by apohl79** [reply]
You need 1.2 to make it work behind a router, as the connection handshake prior to 1.2 requires the plugins to actively open ports.

Also AG is using multiple ports dynamically. You need to open 55056 + serverId and 55088 - 55088 + (number of ag plugins).

**Reply by WilliamKWusik** [reply]
Oh, thank you so much! <3

**Comment by WilliamKWusik** [comment]
Question: when running in a local network, using Gigabit, will this switch work? Switch TP-Link TL-SG105E

I have my internet router, but I guess I could connected the above to the router, and the 2 computers into the switch above, right? It has been a while since I have done any local networking. ;-)

Thanks again! <hug>

**Comment by CaBuffalo** [comment]
Yes, that should work.

On Thu, Jul 29, 2021 at 2:52 PM Wusik Dot Com ***@***.***>
wrote:

> Question: when running in a local network, using Gigabit, will this switch
> work? Switch TP-Link TL-SG105E
>
> I have my internet router, but I guess I could connected the above to the
> router, and the 2 computers into the switch above, right? It has been a
> while since I have done any local networking. ;-)
>
> Thanks again!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/670#discussioncomment-1089018>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AKWS4TBWFWQPZE67TT2HEQBANCNFSM5BA6DCLA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by WilliamKWusik** [comment]
I'm having a hard time opening the 55056 and other ports on both machines. :-o Oh boy... I have asked the guys from Shells.com for some help, maybe I can get it working. I already added all ports to the Windows Firewall. But I think I will need to talk with my ISP tomorrow and check if my fiber modem/router is not blocking anything. I have added to my main router the ports correctly, but nothing. It won't connect. And if I use those online port-open checkers, it tells me both machines don't have 55056 open. :-o Gaaaahhhh...

**Comment by WilliamKWusik** [comment]
![2021-08-01 13_42_20-Window](https://user-images.githubusercontent.com/46165527/127779132-e7629ee2-ff28-40e0-b398-125d89de76c3.png)
![2021-08-01 13_41_57-Window](https://user-images.githubusercontent.com/46165527/127779133-577838cc-b502-4e1e-8148-fd83d62706b1.png)

**Comment by apohl79** [comment]
As AG does not implement any security measures, it might be a good idea to try using it through a vpn tunnel anyway. So maybe thats a way forward.

**Reply by juanriccio** [reply]
I'm not planning to use a "remote" (out-of-LAN)  setup for the time being, but maybe it could be useful to give a look into uPNP or some similar automatic port opening protocol, at least for the local side?

**Comment by Michelobb** [comment]
Hi,
A good alternative could be using ZEROTIER, which simulates a LAN on the Internet, without opening any port on your router.  It creates virtual network adapter on your machine, and you manage potentially a huge network, just as if it was a LAN.
I'm using this to control IP-cams.
Cheers,
Michel

**Reply by WilliamKWusik** [reply]
Wow, thank you so much for the ZeroTier idea! I was able to connect the two computers using it. :-) Now I need to test Latency and other stuff. To see how many plugins my 5 Meg (Upload) 80 Meg (download) internet can do. Soon I will do a video on the setup. :-)
